src/qae/initial-properties.d.ts

```diff
@@ -1,32 +1,35 @@
-declare module "initial-properties";
+import { Component } from "../types/QIX";
```

Comment:
`Component` does not exist. You need to add it

Reply:
I actually followed what we do in table side: https://github.com/qlik-oss/sn-table/blob/main/src/qae/object-properties.d.ts#L1

is that one correct or should I declare something like: 
```ts
declare const RowHeight {
  linesCount: number;
}

declare const Component {
  key: "general" | "theme";
  rowHeight?: RowHeight;
}
```

Reply:
Actually, any declarations in `*.d.ts` file is only for our local development, for our project and internal usage of typescript to not make it confuse with types. 
So the thing is `Component` interface is already declared inside of `types/QIX` and if we are using this as internal (more) type interference of typescript, it makes sense to give it what it needs (import the `Component` interface and use it in `properties.component`)

Reply:
Looks like I was on a branch that was not up-to-date with `main` so I did not have `Component` defined